### PR TITLE
Backport Fixes for react 0.69.X on branch react-native-video 5.2

### PR DESCRIPTION
**Pull request by freeboub**
#### Describe the changes

* Backport patches to support react 0.69 release
* minor upgrade of exoplayer version
* Fix build with native player integration

### CODE REVIEW

**Review by ckaznable — APPROVED** [review]
Working on 0.69.4

**Comment by rafaelbrun** [comment]
Tested on 0.69.4 and it's working :heavy_check_mark:

**Review by rafaelbrun — APPROVED** [review]

**Comment by carl0shd** [comment]
+1 we need this update ASAP 🙏 , tested in 0.69.4

**Comment by subhranshudas** [comment]
When is this going in??

**Comment by freeboub** [comment]
Should done during next week

File: android/build.gradle [hunk 1]
```diff
@@ -22,5 +22,5 @@ android {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
-    implementation 'com.yqritc:android-scalablevideoview:1.0.4'
+    implementation 'com.github.adityaxjha:Android-ScalableVideoView:10e7b4da8b'
```
Comment (CDBridger) [inline android/build.gradle:25]: what is this?

**Comment by egealpay** [comment]
This pull request fixes the problem #2818 that I was having.

**Comment by lyondeveloper** [comment]
Can we please merge this? a real world app is depending on this issue, not working the alpha branch.

**Review by NiharR27 — APPROVED** [review]

**Review by lyondeveloper — APPROVED** [review]
lgtm!

**Comment by freeboub** [comment]
This PR is now  branch 5.2.X, let's close it

**Pull request closed without merging**